### ISSUE TITLE
Error while uploading media in comments 

### ISSUE THREAD

**Issue by pooja-muchandikar** [rtMediaWP/rtMedia#1536]
```
Test Environment
WordPress: v5.3.2
BuddyPress: v5.1.2
rtMedia: v4.5.11
Theme: Kelo theme
```

1. Go to media lightbox
2. Go to comments section 
3. Upload image to comments, observe that on uploading image error is shown in the console and the comment button gets disabled

Refer -> https://drive.google.com/file/d/12PE3jeYRN6Qwcl_tc3RDo75NsNwIPX6u/view

**Comment by dishitpala** [comment]
Not able to understand the issue
With default WordPress theme the above issue is not happening

If we use the Kelo theme, it shows jquery error ( $ is not a function ) if we replace `$` with `jQuery` then it works fine.

Question: Why it's not showing javascript error in 2020 and 2019 theme and shows only in kelo theme
CC @thrijith

**Comment by dishitpala** [comment]
Discussion with @thrijith 
- Need to check `rtMedia.backbone.js` file, where `$` is used even if it is not available and use `jQuery` instead of `$` 
- Need re-factor to avoid bugs in functionality.
- As of now we just need to fix the errors which are shown in the video in the description

CC @pooja-muchandikar

**Comment by pooja-muchandikar** [comment]
@dishitpala 

Please check the fatal errors mentioned in this ticket -https://rtmedia.io/ticket/2560643/ check the screenshot.

**Comment by pooja-muchandikar** [comment]
@dishitpala 

Tested with `pre-release` branch and the JS error shown while uploading media in comments is fixed and now able to upload media in the comments. :heavy_check_mark: 
```
Test Environment
WordPress: v5.3.2
BuddyPress: v5.1.2
rtMedia: v4.5.11 pre-release branch
Theme: Kelo theme, Default 2020 and 2019 theme
```

**Comment by pooja-muchandikar** [comment]
Tested with `cheery-pick/4.5.12` branch and the fix added works as expected.

**Comment by pooja-muchandikar** [comment]
@SID177 

While testing this issue with `develop` branch found an JS error while media upload on commenting. 

Sometime able to upload media to comment and sometimes its showing the JS error. Please check this. 

![Screenshot from 2020-02-14 11-39-22](https://user-images.githubusercontent.com/41000648/74506703-544b7300-4f20-11ea-81b0-f14fe0ee072e.png)

This is specifically happening only with `Kleo` theme.

**Comment by SID177** [comment]
PRs: https://github.com/rtMediaWP/rtMedia/pull/1539, https://github.com/rtMediaWP/rtMedia/pull/1559

**Comment by pooja-muchandikar** [comment]
@SID177 

Tested with `develop` branch with Kleo theme and fix added works fine, no errors are shown and also able to upload media in comment

```
Test Environment
WordPress: v5.3.2
BuddyPress: v5.1.2
rtMedia: v4.5.11 develop branch
Theme: Kelo theme, Default 2020 and 2019 theme
```

**Comment by pooja-muchandikar** [comment]
Tested with `master` branch and fix added works as expected.

**Comment by pooja-muchandikar** [comment]
Released with `v4.6.0`
